[2022-03-14 11:52:01,008] {taskinstance.py:1026} INFO - Dependencies not met for <TaskInstance: tutorial.xsxsxqc.xsxsxqc_get_data scheduled__2022-03-07T10:29:52.784989+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2022-03-14 11:52:01,043] {local_task_job.py:99} INFO - Task is not able to be run
